packages/d/discuss.json

```
@@ -8,28 +8,35 @@
     "vue"
   ],
   "license": "MIT",
-  "homepage": "https://github.com/Lete114/Discuss#readme",
+  "homepage": "https://discuss.js.org",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Lete114/Discuss.git"
+    "url": "git+https://github.com/discussjs/discuss.git"
   },
   "authors": [
     {
       "name": "Lete乐特",
       "url": "https://www.imlete.cn"
     }
   ],
+  "optimization": {
+    "js": false,
+    "css": false,
+    "png": false,
+    "jpg": false
+  },
   "autoupdate": {
     "source": "npm",
     "target": "discuss",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "",
         "files": [
-          "*.js"
+          "dist/*",
+          "assets/*"
```

```suggestion
          "assets/@(emot|img)/*.png",
          "assets/svg/*.svg"
```
I don't see any files in the top-level assets directory, I suspect you need a more specific glob here to match the files you actually want.